### ISSUE TITLE
Subject and text does not work 

### ISSUE THREAD

**Issue by vivchar** [SuddenH4X/awesome-app-rating#7]
Hi

I try to use your library, but when I try to send mail feedback the subject and the text  didn't populate.

It happens when I select the Gmail app, when I select the Email App it works fine

![image](https://user-images.githubusercontent.com/20920610/76840942-98f16380-686a-11ea-9db3-ab68be4b536b.png)

**Comment by SuddenH4X** [comment]
Hi,

thanks for reporting this. I will look into the issue later this week.

**Comment by SuddenH4X** [comment]
Hi @vivchar,

I can reproduce your issue. The Gmail app seems to have changed its implementation recently while other mail apps are still working. I will fix the issue in my next big release (2.0.0).

Meanwhile you can simply overwrite the click listener of the mail feedback button with the following line:
`.setMailFeedbackButton(mailFeedbackButtonTextId: Int, mailFeedbackButtonClickListener: RateDialogClickListener)`

The click listener should then use a working implementation. This one seems to work correctly:
```kotlin
val mailIntent: Intent = Intent(Intent.ACTION_SENDTO).apply {
            data = Uri.parse("mailto:")
            putExtra(Intent.EXTRA_EMAIL, arrayOf("test@example.com"))
            putExtra(Intent.EXTRA_SUBJECT, "subject")
            putExtra(Intent.EXTRA_TEXT, "body")
        }

        if (mailIntent.resolveActivity(packageManager) != null) {
            startActivity(mailIntent)
        } else {
            // no mail app installed
        }
```
